openeo/rest/auth/testing.py

```diff
@@ -83,6 +85,12 @@ def __init__(
             self.requests_mock.post(
                 self.device_code_endpoint, text=self.device_code_callback
             )
+            if self.use_verification_uri_complete:
+                self._set_user_code()
```

Comment:
I think conceptually we can not set the user code here yet, it should be done in the `device_code_endpoint` callback

Reply:
I agree. Was not ideal and was only needed for the mock that is now removed.